com.unity.shadergraph/Editor/Data/Nodes/Utility/SubGraphNode.cs

```diff
@@ -352,6 +352,14 @@ public virtual void UpdateSlots()
                 // Copy defaults
                 switch (prop.concreteShaderValueType)
                 {
+                    case ConcreteSlotValueType.SamplerState:
```

Comment:
As with the other port of these changes, I suspect your formatting error is this indentation (needs to come left one stop).

Reply:
Yup, that was it!  Fixed.